libs/backend/src/websocket/on-close.ts

```diff
@@ -0,0 +1,21 @@
+import { WebSocket } from "@fastify/websocket";
+import { removePlayerFromActivePlayers } from "./remove-player-from-active-players.js";
+
+export function onClose({
+	activePlayers,
+	playerSocketToRemove,
+	code,
+	reason
+}: {
+	activePlayers: WebSocket[];
+	playerSocketToRemove: WebSocket;
+	code: number;
+	reason: Buffer;
+}) {
+	removePlayerFromActivePlayers({
+		activePlayers,
+		playerSocketToRemove
+	});
+
```

Comment:
also loop over all games to check whether the user was in a game / whether the game can be deleted